realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/AuthTests.java

```diff
@@ -827,12 +827,14 @@ public void onError(ObjectServerError error) {
 
     @Test
     @RunTestInLooperThread
-    @Ignore("Depends on https://github.com/realm/realm-java/pull/5909")
+    @Ignore("{\"type\":\"https://docs.realm.io/server/troubleshoot/errors#not-enabled\",\"title\":\"The server was not configured " +
```

Comment:
What does this ignore mean? Same for the others here

Reply:
I just updated the ignore with the specific error message from ROS to have more context (i.e need to configure a password provider)